config.md

```diff
@@ -0,0 +1,164 @@
+# The js-ipfs config file
+
+The js-ipfs config file is a JSON document located in the root directory of the js-ipfs repository.
+
+## Table of Contents
+
+- [`Addresses`](#addresses)
+- [`Bootstrap`](#bootstrap)
+- [`Datastore`](#datastore)
+- [`Discovery`](#discovery)
+- [`Identity`](#identity)
+- [`Swarm`](#swarm)
+
+
+## `Addresses`
+Contains information about various listener addresses to be used by this node.
+
+- `API`
+The IPFS daemon exposes an HTTP API that allows to control the node and
```

Comment:
For all text in this document under a bullet point, can it please be indented appropriately so that when rendered it all lines up vertically?

Text should line up with red line:

![Screenshot 2019-07-17 at 08 45 47](https://user-images.githubusercontent.com/152863/61357098-92d85c80-a86f-11e9-9263-a3b25a765f04.png)